openvdb/openvdb/tree/LeafManager.h

```diff
@@ -231,17 +227,10 @@ class LeafManager
         , mLeafs(other.mLeafs)
         , mAuxBuffers(other.mAuxBuffers)
         , mTask(other.mTask)
-        , mIsMaster(false)
     {
     }
 
-    virtual ~LeafManager()
-    {
-        if (mIsMaster) {
-            delete [] mLeafs;
-            delete [] mAuxBuffers;
-        }
-    }
+    virtual ~LeafManager() { }
```

Comment:
Unrelated to your PR, but why is this even virtual

Reply:
Good question, I'm not sure. Perhaps I can even remove it here.